packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -86,6 +86,8 @@ class PaginatedDataTable extends StatefulWidget {
     this.arrowHeadColor,
     required this.source,
     this.checkboxHorizontalMargin,
+    this.controller,
+    this.primary=false,
```

Comment:
Apologies for the churn, this should probably be null by default. Eventually SingleChildScrollView makes a default decision on primary if it is not set.